WooCommerce/NotificationExtension/OrderNotificationDataService.swift

```diff
@@ -96,6 +96,7 @@ final class OrderNotificationDataService {
             ordersRemote.loadOrder(for: Int64(siteID), orderID: Int64(orderID)) { order, error in
                 switch (order, error) {
                 case (let order?, nil):
+                    MarkOrderAsReadUseCase.markOrderNoteAsReadIfNeeded(network: self.network, noteID: notification.noteID, orderID: orderID)
```

Comment:
I don't think we should mark the notification as read when `quick previewing` the notification on the phone.
I think we should only do it when presenting the order detail in both the app (already done) and in the watch (this update).

And if we only do it on the watch, then it is probably safer to do when presenting the order on `OrderDetailLoaderViewModel/OrderDetailLoaderViewModel` 

What do you think?

Reply:
Hm, could point! I was also having some dilemmas where to call this for this PR, was not sure 100% where to put it. But you have a good point!